test/integration/utility.h

```diff
@@ -110,10 +110,10 @@ class ConnectionStatusCallbacks : public Network::ConnectionCallbacks {
   bool closed() const { return closed_; }
 
   // Network::ConnectionCallbacks
-  void onEvent(uint32_t events) override {
-    closed_ |= (events & Network::ConnectionEvent::RemoteClose ||
-                events & Network::ConnectionEvent::LocalClose);
-    connected_ |= events & Network::ConnectionEvent::Connected;
+  void onEvent(Network::ConnectionEvent event) override {
+    closed_ |= (event == Network::ConnectionEvent::RemoteClose ||
+                event == Network::ConnectionEvent::LocalClose);
+    connected_ |= (event == Network::ConnectionEvent::Connected);
```

Comment:
I think it's fine if you're using the least significant bit as the boolean value (and are happy with some implicit boolean/integer casting going on under the hood).